src/context.rs

```diff
@@ -351,15 +351,22 @@ impl<'buf> Secp256k1<AllPreallocated<'buf>> {
 
     /// Creates a context from a raw context.
     ///
+    /// The returned [`core::mem::ManuallyDrop`] context will never deallocate the memory pointed to
+    /// by `raw_ctx`, nor destroy the context, this may lead to memory leaks. `ManuallyDrop::drop`
```

Comment:
In 382206f0372130d407c29ca3221878dcfc47495e:

nit: `this may lead to memory leaks` should be its own sentence.